cloud/jenkins/psmdb_operator_gke_latest.groovy

```diff
@@ -96,16 +96,24 @@ EOF
     }
     echo "USED_PLATFORM_VER=$USED_PLATFORM_VER"
 
+    if ("$ARCH" == "amd64") {
```

Comment:
Can we make it with backward compatibility? If arch is arm64 - use t2a-standard-4, else use amd one?